### PR TITLE
Coverage search check for Encounter.read scope while searching by -encounter

### PR DESCRIPTION
**Pull request by SantanuBanerjee**
Before change:
Screenshot 2020-10-07 at 7 42 02 PM

After change:
Screenshot 2020-10-07 at 7 48 03 PM

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/financial/coverage.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ Search for Patient-level or Encounter-level Coverages that meet supplied query p
 _Implementation Notes_
   
 * Public Healthcare represents an insurance policy funded by a public health system such as a provincial or national health plan.
+* For Coverage search by -encounter, both Encounter.read and Coverage.read scopes are required
```
Comment (sydneanschutz) [inline content/millennium/r4/financial/coverage.md:56]: I think Encounter.read and Coverage.read should be in quotes like seen here: https://github.com/cerner/fhir.cerner.com/blame/main/content/millennium/r4/other/binary.md#L45

Comment (SantanuBanerjee) [reply content/millennium/r4/financial/coverage.md]: yeah, that's correct.

Comment (SantanuBanerjee) [reply content/millennium/r4/financial/coverage.md]: Screenshot 2020-10-08 at 3 48 55 PM
After the change.

**Review by sydneanschutz — APPROVED** [review]

File: content/millennium/r4/financial/coverage.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ Search for Patient-level or Encounter-level Coverages that meet supplied query p
 _Implementation Notes_
   
 * Public Healthcare represents an insurance policy funded by a public health system such as a provincial or national health plan.
+* For Coverage search by -encounter, both `Encounter.read` and `Coverage.read` scopes are required
```
Comment (exopsRamba) [inline content/millennium/r4/financial/coverage.md:56]: nit: can we wrap '-encounter' in back tics too since it's a search param, i.e. `-encounter`

**Comment by SantanuBanerjee** [comment]
The change is not required after discussion, as this is handled in some different way from the local service.

**Comment by MaxPhilips** [comment]
@SantanuBanerjee i'm going to delete this branch since we decided this change was not needed

**Comment by SantanuBanerjee** [comment]
sure Max. Thanks.

**Pull request closed without merging**